exec/mem.go

```diff
@@ -135,18 +141,25 @@ func (ce *memExecutor) Exec(uid string, ctx context.Context, model *spec.ExpMode
 			return spec.ReturnFail(spec.Code[spec.IllegalParameters],
 				"--mem-percent value must be a prositive integer and not bigger than 100")
 		}
+	} else if memReserveStr != "" {
+		memReserve, err = strconv.Atoi(memReserveStr)
+		if err != nil {
+			return spec.ReturnFail(spec.Code[spec.IllegalParameters],
+				"--reserve value must be a positive integer")
+		}
 	} else {
-		memPercent = 100
```

Comment:
If you did not add flag before, the default memory is 100%. Now you must add `--mem-percent` or `--reserve`. Is it necessary to consider compatibility problem?

Reply:
yes,  we should give the default vaule.